fastly/block_fastly_service_product_enablement.go

```diff
@@ -71,6 +71,11 @@ func (h *ProductEnablementServiceAttributeHandler) GetSchema() *schema.Schema {
 			Optional:    true,
 			Description: "Enable Image Optimizer support (all backends must have a `shield` attribute)",
 		}
+		blockAttributes["ngwaf"] = &schema.Schema{
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Description: "Enable NGWAF support",
```

Comment:
I would recommend using Product name "Next-Gen WAF" instead of the abbreviation.